.github/workflows/release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  push:
```

Comment:
It would be nice to also make a release and publish to pypi when someone pushes a tag, like here: https://github.com/BlueBrain/Currentscape/blob/main/.github/workflows/build.yml#L7